packages/xrpl/src/Wallet/fundWallet.ts

```diff
@@ -179,6 +221,8 @@ async function onEnd(
       resolve,
       reject,
     )
+  } else if (client.url === 'wss://hooks-testnet-v2.xrpl-labs.com') {
```

Comment:
I don't think this would work for the JSON client because it doesn't use the `wss`, it uses `http`/`https`. Potentially look for `client.url.contains('hooks-testnet-v2.xrpl-labs.com')` (Not sure if contains is valid js, but that's the idea)